js/components/graph/GraphDropdown.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
```

Comment:
This entire file seems to be using tabs instead of spaces. Make sure to configure your text editor to use spaces, and then convert the indentation in this file.

Reply:
The same is true to the GraphDropdown test file below.